scripts/package

```diff
@@ -4,6 +4,24 @@ set -eu
 # adapted from https://github.com/johannes-wolf/cetz/blob/35c0868378cea5ad323cc0d9c2f76de8ed9ba5bd/scripts/package
 # licensed under Apache License 2.0
 
+. "$(dirname "${BASH_SOURCE[0]}")/setup"
+
+if (( $# < 1 )) || [[ "${1:-}" == "help" ]]; then
```

Comment:
I can't say I know what this does, I assume it checks for the length of the argument array of this script and displays help if it's 0 or contains "help", but surely there are less terse ways to write this.

Readability is very important for scripts, a script should generally strive towards using long option names here they exist and use non-short hand syntax.

The same goes for the uninstalling script.